plugins/GCodeProfileReader/GCodeProfileReader.py

```diff
@@ -72,7 +72,7 @@ def read(self, file_name):
         Logger.log("i", "Serialized the following from %s: %s" %(file_name, repr(serialized)))
 
         # Create an empty profile with the name of the G-code file
-        profile = InstanceContainer(os.path.basename(os.path.splitext(file_name)[0]))
+        profile = InstanceContainer("G-code-imported-profile")
```

Comment:
Why do you change this functionality? Now it behaves differently from importing a normal profile.
See https://github.com/Ultimaker/Cura/blob/master/cura/CuraContainerRegistry.py#L144
Also it is not translated.